config/migrator/crosswalk.yml

```diff
@@ -0,0 +1,506 @@
+crosswalk:
+  - od2_property: alternative
```

Comment:
We chatted about it briefly, but i think `od2_` should be removed from all of the configuration/code in the PR just to keep this a little more generic.